eng/packages/http-client-csharp-mgmt/generator/TestProjects/Spector/Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <IsClientLibrary>true</IsClientLibrary>
+    <IsTestProject>false</IsTestProject>
```

Comment:
we can just rename `TestProjects.Spector.Tests.csproj` to start with `Azure.Generator.`, so that it will be client library and not a test project.